src/modules/meals/counter.js

```diff
@@ -0,0 +1,18 @@
+export const countMeals = (meals) => {
+  const mealsCount = meals ? meals.length : 0;
+  return mealsCount;
+};
```

Comment:
- [ ] Please make sure that this method is implemented using [this](https://github.com/dennis-every/dd_eats/issues/6#:~:text=A%20counter%20function%20should%20look%20for%20specific%20DOM%20elements%20(e.g.%20for%20the%20comments%20counter%20it%20should%20look%20for%20comments)%20and%20make%20the%20counting%20based%20on%20what%20is%20actually%20displayed%20on%20the%20page.) description which is by counting the DOM elements and not by using the fetched items from the API directly. (Also please make sure to also account for the tests)